src/main/java/ch/njol/skript/classes/data/BukkitEventValues.java

```diff
@@ -238,6 +238,13 @@ public Player get(final BlockPlaceEvent e) {
 				return e.getPlayer();
 			}
 		}, 0);
+		EventValues.registerEventValue(BlockPlaceEvent.class, ItemStack.class, new Getter<ItemStack, BlockPlaceEvent>() {
+			@Override
+			@Nullable
+			public ItemStack get(BlockPlaceEvent event) {
+				return event.getItemInHand();
+			}
+		}, EventValues.TIME_NOW);
```

Comment:
We should add timestates to this event value, e.g. `past event-item` should contain the amount before subtracting 1 (if not creative mode), and `future` should have 1 less (if not creative).

Reply:
I presume the present state should stay as is, with the amount before subtracting?

Reply:
Yes